lib/model.js

```diff
@@ -191,19 +191,34 @@ class Model {
       self._expandAttributes(options);
     }
 
-    if (!options.include) {
-      return;
+    if (options.include) {
```

Comment:
Both include and cte preprocessing is identical, except for method call in end

what about ```['cte', 'include'].forEach```